src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileServiceImpl.java

```diff
@@ -64,6 +91,24 @@ public ConcurRequestExtractFileValidationService getConcurRequestExtractFileVali
         return this.concurRequestExtractFileValidationService;
     }
 
+    private String parseRequestExtractFileNameFrom(String requestExtractFullyQualifiedFileName) {
+        //plus 1 because we do not want the forward slash
+        return requestExtractFullyQualifiedFileName.substring((requestExtractFullyQualifiedFileName.lastIndexOf(ConcurConstants.FORWARD_SLASH)) + 1);
```

Comment:
Ok, I think I understand now. You want to retrieve just the file name from the full path string.